### PR TITLE
feat(hero-product): add hero-product

### PR DESCRIPTION
**Pull request by zwaardje**
Add hero product
Add product

### CODE REVIEW

File: src/js/components/molecules/product/stories/product.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Product
```
Comment (rhelling) [inline src/js/components/molecules/product/stories/product.md]: Description and links to DSM documentation are missing

File: src/js/components/molecules/product/stories/product.stories.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    return () => {
+        const knobs = getKnobs(
+            data[type],
+            knobsFormat,
+        );
+
+        return {
+            data() {
+                return {
+                    slot: knobs.slot,
```
Comment (rhelling) [inline src/js/components/molecules/product/stories/product.stories.ts]: Can be removed

File: src/js/components/organisms/hero-product/hero-product.scss [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    @extend %reset-last-margin;
+
+    .hero__container {
+        @include max-mobile() {
+            padding-top: $buildingUnit;
+        }
+    }
+
+    .hero__image--basic,
+    .hero__image--product{
```
Comment (rhelling) [inline src/js/components/organisms/hero-product/hero-product.scss]: It shouldn't be necessary to wrap a modifier(hero__image--product) in a modifier (hero--product)

File: src/js/components/organisms/hero-product/hero-product.scss [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+            margin-bottom: 0;
+        
+            &--multiple {
+                border-right: 1px solid $colorGray;
+            }
+        }
+    }
+
+    .hero__button {
+        display: table;
```
Comment (rhelling) [inline src/js/components/organisms/hero-product/hero-product.scss]: Do you need some help with this?

File: src/js/components/organisms/hero-product/hero-product.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+import { HeroProductProps } from './hero-product.models';
+import './hero-product.scss';
+import productKnobsFormat from '../../molecules/product/stories/product.knobs-format';
```
Comment (rhelling) [inline src/js/components/organisms/hero-product/hero-product.ts]: This import is unused

File: src/js/components/organisms/hero-product/hero-product.unit.spec.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+            const mount = build(HeroProduct, { props: {}});
+
+            it('should render an HeroProduct component', () => {
+                const vm = mount();
+                expect(vm.$el).toBeDefined();
+                vm.$destroy();
+            });
+        });
+    });
+});
```
Comment (rhelling) [inline src/js/components/organisms/hero-product/hero-product.unit.spec.ts]: Can you add some more meaningful tests?

File: src/js/components/organisms/hero-product/stories/hero-product.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+# Hero Product
+
+## DSM
+* [organisms/hero-product](https://ultimaker.invisionapp.com/dsm/ultimaker/ultimaker-com/asset/components/)
```
Comment (rhelling) [inline src/js/components/organisms/hero-product/stories/hero-product.md]: Documentation is missing

File: src/js/components/organisms/hero-product/stories/hero-product.stories.ts [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+const getStory = (type) => {
+    return () => {
+        const knobs = getKnobs(
+            data[type],
+            knobsFormat,
+        );
+
+        console.log(knobs);
+
+        console.log(data);
```
Comment (rhelling) [inline src/js/components/organisms/hero-product/stories/hero-product.stories.ts]: Debugging stuff can be removed

File: src/js/components/organisms/hero-product/stories/hero-product.stories.ts [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+        return {
+            data() {
+                return {
+                    slot: knobs.slot,
```
Comment (rhelling) [inline src/js/components/organisms/hero-product/stories/hero-product.stories.ts]: This is not used

File: src/js/components/organisms/hero-product/hero-product.ts (hunk 5 above)
Comment (RobinDevelopersNL) [reply src/js/components/organisms/hero-product/hero-product.ts]: fixed

File: src/js/components/organisms/hero-product/hero-product.scss (hunk 4 above)
Comment (RobinDevelopersNL) [reply src/js/components/organisms/hero-product/hero-product.scss]: Is gone

File: src/js/components/organisms/hero-product/hero-product.scss (hunk 3 above)
Comment (RobinDevelopersNL) [reply src/js/components/organisms/hero-product/hero-product.scss]: Fixed

File: src/js/components/molecules/product/stories/product.stories.ts (hunk 2 above)
Comment (RobinDevelopersNL) [reply src/js/components/molecules/product/stories/product.stories.ts]: Fixed

File: src/js/components/organisms/hero-product/stories/hero-product.stories.ts (hunk 8 above)
Comment (RobinDevelopersNL) [reply src/js/components/organisms/hero-product/stories/hero-product.stories.ts]: Fixed

File: src/js/components/organisms/hero-product/stories/hero-product.stories.ts (hunk 9 above)
Comment (RobinDevelopersNL) [reply src/js/components/organisms/hero-product/stories/hero-product.stories.ts]: Fixed

File: src/js/components/molecules/product/stories/product.md (hunk 1 above)
Comment (RobinDevelopersNL) [reply src/js/components/molecules/product/stories/product.md]: Fixed

File: src/js/components/organisms/hero-product/hero-product.unit.spec.ts (hunk 6 above)
Comment (RobinDevelopersNL) [reply src/js/components/organisms/hero-product/hero-product.unit.spec.ts]: Fixed

File: src/js/components/organisms/hero-product/stories/hero-product.md (hunk 7 above)
Comment (RobinDevelopersNL) [reply src/js/components/organisms/hero-product/stories/hero-product.md]: Fixed

**Pull request merged**